draft-packages/card/KaizenDraft/Card/Card.tsx

```diff
@@ -1,32 +1,58 @@
 import * as React from "react"
+import cx from "classnames"
 import styles from "./styles.module.scss"
 
+export type CardVariants =
```

Comment:
Update to use `mood` prop name rather than variant.

Reply:
Just something I've learnt recently that I don't think was clear initially

Reply:
I might split this into a seperate small task since we should also change the name of `variant` prop, which would cause a breaking change :)